python/TestHarness/util.py

```diff
@@ -529,6 +529,9 @@ def getCapabilities(exe):
     """
     assert exe
     output = runCommand("%s --show-capabilities" % exe)
+    if output.startswith("ERROR"):
+        print("Error! Executable crashed running --show-capbilities.")
+        exit(1)
```

Comment:
No, that's not right. You might get a crash without the `ERROR` string.

Reply:
This is right, you get an ERROR whenever the executable crashes.

line 226 appends ERROR when the executable crashes.
```
    p = subprocess.Popen(cmd, cwd=cwd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, close_fds=should_close, shell=True)
    output = p.communicate()[0].decode('utf-8')
    if (p.returncode != 0):
        output = 'ERROR: ' + output
```
I can't change the code in the runCommand function because its called by the testHarness and shouldn't exit if it segfaults, it should return an ERROR.